### ISSUE TITLE
missing shared libraries: libnuma.so

### ISSUE THREAD

**Issue by slashmili** [hhvm/packaging#144]
I'm testing on `Ubuntu 15.10`, after installing hhvm package version `3.14.1~wil` I get error related to missing shared library

```
$ hhvm --version
hhvm: error while loading shared libraries: libnuma.so.1: cannot open shared object file: No such file or directory
```

after installing `libnuma-dev` hhvm works fine. Is it missing on debian package's dependency list?

**Comment by Orvid** [comment]
It looks like it, but that's really odd, as I didn't think we supported using Numa on the open source build (even odder is that it worked when I tried it before I published the package, as well as that it's taken a week since after I re-packaged everything for 3.14.1 for anyone to have any issues O_o...)

As there's apparently a similar issue with it on 16.04, I'll do a bit of digging and then re-package 15.10 and 16.04.

**Comment by Orvid** [comment]
This should be fixed with the 3.14.2 packages, we'll look at re-enabling numa support on the OSS builds for 3.16, but, as 3.15 will be an LTS release, we don't want to go trying to enable it, as it hasn't really been tested on the OSS build.
